integrations-and-supported-tools/matplotlib/scripts/requirements.txt

```diff
@@ -1,3 +1,3 @@
-matplotlib==3.2.2
-plotly==4.4.1
-neptune-client==0.9.5
\ No newline at end of file
+matplotlib
```

Comment:
@SiddhantSadangi the requirements here should have explicit versions so that people know what works.
We'll be updating versions periodically.